src/tests/control/test_events.py

```diff
@@ -683,8 +684,13 @@ def test_delete(self):
         doc = self.get_doc('/control/event/ccc/30c3/subevents/%d/delete' % self.subevent1.pk)
         assert doc.select("button")
         doc = self.post_doc('/control/event/ccc/30c3/subevents/%d/delete' % self.subevent1.pk, {})
-        assert doc.select(".alert-success")
+        if self.event1.subevents.count() > 1:  # we created 2 subevents. Number of subevents left after one is deleted
```

Comment:
What is this if statement doing here? This should always be the case, so its unnecessary. Even worse: If it is *not* true, the further checks are skipped and the test will pass! Better use ``assert self.event1.subevents.count() > 1`` if you really want to check this instead :)

Reply:
Yeah, you are right, since we are creating 2 subevents, this would always be true. Will fix it soon,